imports/methods/createChatImageUpload.ts

```diff
@@ -0,0 +1,7 @@
+import TypedMethod from "./TypedMethod";
+
+export default new TypedMethod<
+  { puzzleId: string; mimeType: string },
+  | { publicUrl: string; uploadUrl: string; fields: Record<string, string> }
+  | undefined
```

Comment:
This disjunction doesn't actually match the type that the method is checking for, right? Should just be the first option?